docs/ja-jp/installation.md

```diff
@@ -1,10 +1,10 @@
-# Atomへのインストール
+# Atom へのインストール
```

Comment:
This diff seems to be caused by prettier 🤔

Reply:
I think it's fine :)